tests/integration-test/src/tests/utils.rs

```diff
@@ -49,13 +49,25 @@ pub fn add_xdp_pass(iface: &str, priority: u32) -> Result<String> {
     let output = Command::cargo_bin("bpfctl")?
         .args([
             "load-from-image",
+            "--global",
```

Comment:
Can we make sure we still have a follow up issue for an E2E test with the global data + proceed_on features that actually loads AND verifies them?

Reply:
I think that's covered in https://github.com/bpfd-dev/bpfd/issues/394.